src/test/java/statisticker/StatisticsTest.java

```diff
@@ -38,5 +38,34 @@ public void reportsNaNForEmptyInput()
         
     }
     
+    @Test
+    public void reportwhenlistcontainsNaNasinput()
+    {
```

Comment:
try to include the output in the test name. between the 'report' and the 'when'
Like reportStatsOfRealNumbersWhenListContainsNaNasInput